portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+$(document).ready(initClasses);
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function initClasses() {
+  $(".card").click(openCard);
 }
+
+ function openCard(event){
+   const Card = event.target;
```

Comment:
use lowerCamelCase for variable names. Capitalized ones are reserved for classes.
https://google.github.io/styleguide/jsguide.html#naming-local-variable-names